cypress/e2e/modifiedBoard.cy.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types= "cypress" />
+
+describe('Modify', function(){
```

Comment:
Move all the content of this file to: cypress/e2e/testTrello.cy.js

Reply:
Do it